src/Caching/Hybrid/src/Internal/InbuiltTypeSerializer.cs

```diff
@@ -0,0 +1,54 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Buffers;
+using System.Diagnostics;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Microsoft.Extensions.Caching.Hybrid.Internal;
+internal sealed class InbuiltTypeSerializer : IHybridCacheSerializer<string>, IHybridCacheSerializer<byte[]>
+{
+    private static InbuiltTypeSerializer? _instance;
+    public static InbuiltTypeSerializer Instance => _instance ??= new();
```

Comment:
This doesn't look thread-safe.  Is that important?

Reply:
no, it isn't critical that we get the same one reliably at init - we just want to avoid having eleventy billion